platform/viewer/src/studylist/StudyListWithData.js

```diff
@@ -209,6 +209,23 @@ class StudyListWithData extends Component {
         <ConnectedDicomStorePicker
           isOpen={this.state.modalComponentId === 'DicomStorePicker'}
           onClose={this.closeModals}
+          // server={[
+          //   {
+          //     name: 'prostateImages',
+          //     imageRendering: 'wadors',
+          //     thumbnailRendering: 'wadors',
+          //     qidoSupportsIncludeField: false,
+          //     type: 'dicomWeb',
+          //     qidoRoot:
+          //       'https://healthcare.googleapis.com/v1beta1/projects/dicomserver-249915/locations/northamerica-northeast1/datasets/medicalImages/dicomStores/prostateImages/dicomWeb',
+          //     wadoRoot:
+          //       'https://healthcare.googleapis.com/v1beta1/projects/dicomserver-249915/locations/northamerica-northeast1/datasets/medicalImages/dicomStores/prostateImages/dicomWeb',
+          //     wadoUriRoot:
+          //       'https://healthcare.googleapis.com/v1beta1/projects/dicomserver-249915/locations/northamerica-northeast1/datasets/medicalImages/dicomStores/prostateImages/dicomWeb',
+          //     active: true,
+          //     supportsFuzzyMatching: false,
+          //   },
+          // ]}
```

Comment:
Do we need this as a comment here?

Reply:
I put it there just as a `server` prop example for the PR. But yes, this shouldn't be in the codebase if we'd like to merge, removing it.